### PR TITLE
chore: add editconfig, vscode extension recommendations

### PR DESCRIPTION
**Pull request by phrough**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (ychhabra-eightfold) [inline .editorconfig:1]: We use prettier, please setup prettier on ur editor and this wont be needed, will add this to contribution guidlines to make it clearer for future devs.

File: .vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (ychhabra-eightfold) [inline .vscode/extensions.json:1]: I dont think any editor config/extenstions settings should part of this repo

**Review by ychhabra-eightfold — CHANGES_REQUESTED** [review]

**Comment by thiyagaraj-eightfold** [comment]
Hey @phrough , thanks for the initiative!
Can you create a github issue before creating a PR so that the team can comment and have discussions.

File: .editorconfig (hunk 1 above)
Comment (phrough) [reply .editorconfig:1]: Sounds good. Loosely related, there *might* be an issue in which prettier and eslint are battling each other work during the pre commit hook. I've seen some strange behavior there, but I'm not certain about it.

**Pull request closed without merging**